apps/steam/steam.star

```diff
@@ -0,0 +1,160 @@
+"""
+Applet: Steam
+Summary: Steam Now Playing
+Description: Displays the game that the specified user is currently playing, or the most recent games if currently not in-game.
+Author: Jeremy Tavener
+"""
+
+load("render.star", "render")
+load("http.star", "http")
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("secret.star", "secret")
+load("schema.star", "schema")
+
+API_BASE_URL = "http://api.steampowered.com/"
+API_PLAYER_SUMMARIES = API_BASE_URL + "ISteamUser/GetPlayerSummaries/v0002"
+API_RECENTLY_PLAYED_GAMES = API_BASE_URL + "IPlayerService/GetRecentlyPlayedGames/v0001"
+API_OWNED_GAMES = API_BASE_URL + "IPlayerService/GetOwnedGames/v0001"
+
+def main(config):
+    steam_id = config.get("steam_id", None)
+    api_key = secret.decrypt("""
+        AV6+xWcE+DYxl3gCCIpeRaoTWtnpTqIEqS8zWqnvoStfp/IrLru54NPMHgekh8jwUshSLUFp4Y9GFy7C/
+        zLyBm1DzJODRDfTo3NROgQt7jJI6nzK/8I6MPMv33VZ4/ZjcSH3BkQ0THHB0XcGIv++ntIj3/tcDsvQtr
+        NkoszNXck6JEZaDpo=
```

Comment:
Let's use this API key owned by tidbyt.com. That way we can contact them for higher rate limits and also OAuth support in the future.

```suggestion
        AV6+xWcE9acn7G/Amrp+rzPaSppKSmwWdVBJklcT6q63X9wG8UFn0ma1U9oCtmwjzVyZFF1JzzpLre2K
        mVuRNN+Lk7UICOvuChgDC6pstPv9ecAHFGu9h9C7kc3ntBcbUARJIVWXZVH2ji+PtIis/jekWarIgISt
        qukjL49KF/GT3BtMwJ4=
```

Reply:
Absolutely - much better than using mine!